app/models/grda_warehouse/hud/client.rb

```diff
@@ -1779,13 +1779,13 @@ def self.strict_search(criteria, client_scope: none)
 
       if first_name.present?
         first_name_ids = source.where(
-          nf('LOWER', [arel_table[:FirstName]]).eq(first_name.downcase),
+          nf('TRIM', [nf('LOWER', [arel_table[:FirstName]])]).eq(first_name.downcase),
```

Comment:
Do you think we should call `.strip` on the input as well?  Maybe we do that upstream?  
So you can search "Elliot  "?

Reply:
excellent